test/functional/query-builder/order-by/entity/Post.ts

```diff
@@ -21,4 +22,8 @@ export class Post {
     @Column()
     num2: number = 1;
 
-}
\ No newline at end of file
+    @Column({ type: "geometry" })
+    @Index({ spatial: true })
```

Comment:
This looks like it broke the build; some of the before/after hooks are now hanging. Is there a good way to diagnose this?

Reply:
Ah, it looks like introducing support for spatial indexes in Postgres is conflicts with tests that also execute in MySQL. I think I've just pushed a fix for this.